pod/perlre.pod

```diff
@@ -492,6 +492,10 @@ may be easier to grasp than the squashed equivalents
     /[d-eg-i3-7]/
     /[!@"#$%^&*()=?<>']/
 
+Note that this unfortunately doesn't mean that your bracketed classes
+can extend over multiple lines.  The closing bracket must be on the same
+line as the opening one.
```

Comment:
That reads like it's a syntax error, but it actually just means newline will be one of the characters in the class. I think the second sentence could read something like: "Unless the closing bracket is on the same line as the opening one, the newline character will be part of the class just as if you'd written `C<\n>`." (Sadly that also means we'd have back-compat problems if we wanted it to ignore newlines in the future.)

The placement of the `/xx` docs directly after "A common pitfall is to forget that `"#"` characters begin a comment under `/x` ..." may also explain some of the additional confusion - the parenthetical "... (outside a bracketed character class, which is unaffected by `/x`) ..." a couple of paragraphs earlier can easily get lost here. It's not immediately obvious to me how to rearrange things to minimize the opportunity for such confusion.